### PR TITLE
Add toggle for the island menu

### PR DESCRIPTION
**Pull request by dlsf**

### CODE REVIEW

File: src/main/java/com/iridium/iridiumskyblock/commands/CommandManager.java [hunk 1]
```diff
@@ -107,7 +107,12 @@ public boolean onCommand(@NotNull CommandSender commandSender, org.bukkit.comman
                 User user = IridiumSkyblock.getInstance().getUserManager().getUser(player);
                 Optional<Island> island = user.getIsland();
                 if (island.isPresent()) {
-                    player.openInventory(new InventoryConfigGUI(IridiumSkyblock.getInstance().getInventories().islandMenu).getInventory());
+                    if (IridiumSkyblock.getInstance().getConfiguration().disableMainMenu) {
+                        String command = IridiumSkyblock.getInstance().getCommands().helpCommand.aliases.get(0);
```
Comment (PeachesMLG) [inline src/main/java/com/iridium/iridiumskyblock/commands/CommandManager.java:111]: So I realised Im being really dumb when doing stuff this way, we should just do helpCommand.execute. Idk if u wana put another pr through to do that, if not ill do it

**Review by PeachesMLG — CHANGES_REQUESTED** [review]

File: src/main/java/com/iridium/iridiumskyblock/configs/Configuration.java [hunk 2]
```diff
@@ -38,6 +38,7 @@
     public boolean respawnOnIsland = true;
     public boolean obsidianBucket = true;
     public boolean patreonMessage = true;
+    public boolean disableIslandMenu = false;
```
Comment (PeachesMLG) [inline src/main/java/com/iridium/iridiumskyblock/configs/Configuration.java:41]: also I think this should be the opposite
public boolean islandMenu = true

Comment (PeachesMLG) [reply src/main/java/com/iridium/iridiumskyblock/configs/Configuration.java:41]: so instead of disabling it have the boolean be to enable it and default to true

Comment (dlsf) [reply src/main/java/com/iridium/iridiumskyblock/configs/Configuration.java:41]: Yeah, I was thinking the same, no idea why I didn't do it
